src/Components/Facility/TriageForm.tsx

```diff
@@ -61,6 +61,11 @@ const triageFormReducer = (state = initialState, action: any) => {
   }
 };
 
+const scrollTo = (id: string | boolean) => {
+  const element = document.querySelector(`#${id}`);
+  element?.scrollIntoView({ behavior: "smooth", block: "center" });
+};
```

Comment:
Move this to utils, since we have couple of duplicate implementation of this. Also refactor other usage to point to this reusable utility.